src/vivarium_public_health/metrics/disease.py

```diff
@@ -65,140 +36,62 @@ def __init__(self, disease: str):
     def __repr__(self):
         return f"DiseaseObserver({self.disease})"
 
-    ##########################
-    # Initialization methods #
-    ##########################
-
-    # noinspection PyMethodMayBeStatic
-    def _get_configuration_defaults(self) -> Dict[str, Dict]:
```

Comment:
Do we? Because of the expectation that developers would subclass the observer for the stratifications, I thought it was reasonable to expect them to use the correct `disease` argument in the configuration defaults.

Reply:
Nevermind, I see a few issues...